includes/helper/class-wp-job-manager-helper-api.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * WP_Job_Manager_Helper_API
+ */
+class WP_Job_Manager_Helper_API {
+
+	private static $api_url = 'https://wpjobmanager.com/';
+
+	/**
+	 * Make a licence helper API request.
+	 *
+	 * @param array $args
+	 * @param bool  $return_error
+	 *
+	 * @return array|bool|mixed|object
+	 */
+	private static function request( $args, $return_error = false ) {
+		$defaults = array(
+			'instance'       => self::get_site_url(),
+			'plugin_name'    => '',
+			'version'        => '',
+			'api_product_id' => '',
+			'licence_key'    => '',
+			'email'          => '',
+		);
+
+		$args    = wp_parse_args( $args, $defaults );
+		$request = wp_safe_remote_get( self::$api_url . '?' . http_build_query( $args, '', '&' ), array(
+			'timeout' => 10,
+			'headers' => array(
+				'Accept' => 'application/json',
+			),
+		) );
+
+		if ( is_wp_error( $request ) || 200 !== wp_remote_retrieve_response_code( $request ) ) {
+			if ( $return_error ) {
+				if ( is_wp_error( $request ) ) {
+					return array(
+						'error_code' => $request->get_error_code(),
+						'error' => $request->get_error_message(),
+					);
+				}
+				return array(
+					'error_code' => wp_remote_retrieve_response_code( $request ),
+					'error' => 'Error code: ' . wp_remote_retrieve_response_code( $request ),
+				);
+			}
+			return false;
+		}
+
+		$response = json_decode( wp_remote_retrieve_body( $request ), true );
```

Comment:
It's worthwhile to put a @ before the json_decode(), as a broken response from the server could cause a fatal error otherwise (stupid PHP)